### PR TITLE
Add WDL, READme

### PR DESCRIPTION
**Pull request by hannahdriver**

### CODE REVIEW

File: t1k.wdl [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+  call hlaTyping {
+    input:
+      inputFastq1 = fastqR1,
+      inputFastq2 = fastqR2,
+      preset = resources[libraryDesign].hlaPreset,
+      referenceSequence = resources[libraryDesign].hlaReferenceSequence,
+      outputFileNamePrefix = outputFileNamePrefix
+  }
+
+  if(libraryDesign=="WG") {
```
Comment (gavin-peng) [inline t1k.wdl:47]: Here only one branch for case "WG", the WT resources never used. Don't know if this is by design?
My understanding is wdl version 1.0 has no "else" clause, so needs two "if" for two condition branches.

File: commands.txt [hunk 2]
```diff
@@ -0,0 +1,30 @@
+## Commands
+This section lists command(s) run by the t1k workflow
+
+* Running t1k
+
+=== HLA Typing ===.
+
+<<<
```
Comment (pruzanov) [inline commands.txt]: I would change these to back ticks. It's really optional, but will save your time when upgrading/updating README

File: README.md [hunk 3]
```diff
@@ -1 +1,84 @@
...
+`hlaResults`|File|File with HLA typing results
+`kirResults`|File?|File with KIR typing results
+
+
+## Commands
+ This section lists command(s) run by the t1k workflow
+ 
+ * Running t1k
+ 
+ === HLA Typing ===.
```
Comment (pruzanov) [inline README.md]: This may be changed to ### for pretty printing

File: README.md [hunk 4]
```diff
@@ -1 +1,84 @@
...
+ 
+ <<<
```
Comment (pruzanov) [inline README.md]: back ticks here will improve formatting

**Review by pruzanov — APPROVED** [review]

**Review by gavin-peng — APPROVED** [review]
Looks good

**Pull request merged**